src/service.rs

```diff
@@ -359,64 +359,70 @@ where
         self.service_context.control()
     }
 
-    fn push_back(&mut self, priority: Priority, id: SessionId, event: SessionEvent) {
-        if priority.is_high() {
-            self.high_write_buf.push_back((id, event));
-        } else {
-            self.write_buf.push_back((id, event));
-        }
-    }
-
     #[inline(always)]
-    fn distribute_to_session_process<D: Iterator<Item = (SessionId, SessionEvent)>>(
+    fn distribute_to_session_process(
```

Comment:
Not related to this PR: I think we should limit the number of events to dispatch in one round.

Reply:
In fact, here's what I should say: under low stress, whatever you do is not a problem, but under prolonged high stress, whatever you do is wrong